library/sources/GraphQL.ts

```diff
@@ -119,6 +133,10 @@ export class GraphQL implements Wrapper {
       .withVersion("^16.0.0")
       .onFileRequire("execution/execute.js", (exports, pkgInfo) => {
         this.wrapExecution(exports, pkgInfo);
+      })
+      .onRequire((exports, pkgInfo) => {
+        this.GraphQLError = exports.GraphQLError;
+        this.visitFn = exports.visit;
```

Comment:
Maybe combine them? And keep a reference to the original module? 

Then we can immediately return when execute is called and the module is not available for some reason?